src/components/GrafanaContent.js

```diff
@@ -0,0 +1,24 @@
+import React, {useEffect} from "react";
```

Comment:
this file needs `prettier` ran on it - the linting doesn't seem to match the rest of the app